### PR TITLE
Fix audio group membership

### LINKED ISSUE #673: adding user openhab late in the install process makes us miss some steps ?
just a quick issue to not forget about checking this out
I just noticed running the test suite:

[...]
2019-08-24_08:18:12_UTC [openHABian] Enabling Audio output...
$ adduser openhab audio
adduser: The user `openhab' does not exist.

### PR DESCRIPTION
**Pull request by mstormi**
to fix #673

### CODE REVIEW

File: openhabian-setup.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2034
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: Can this be avoided?

File: openhabian-setup.sh [hunk 2]
```diff
@@ -61,7 +62,6 @@ if [[ -n "$UNATTENDED" ]]; then
   timezone_setting
   locale_setting
   hostname_change
-  if is_pi; then memory_split; enable_rpi_audio; fi
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: Would leave memory_split up here. No real reason, just for the sake of not changing more than meaningful

File: openhabian-setup.sh (hunk 1 above)
Comment (mstormi) [reply openhabian-setup.sh]: yes but then either
 - we need it multiple times 
 - we export these variables which is what I did before. But Elias didn't like it.

File: openhabian-setup.sh (hunk 2 above)
Comment (mstormi) [reply openhabian-setup.sh]: That would mean to clutter functionality.
As memory_split() only writes into /boot/config.txt it does not really matter when it does that.

File: openhabian-setup.sh (hunk 1 above)
Comment (EliasGabrielsson) [reply openhabian-setup.sh]: In this case I would say a global `shellcheck disable=SC2034` is preferred to not clutter the code.

**Comment by EliasGabrielsson** [comment]
LGTM

**Comment by mstormi** [comment]
Agree. I've changed it to match your proposal.

**Comment by mstormi** [comment]
any reason left why not to apply this?

**Comment by BClark09** [comment]
Thanks @mstormi

**Pull request merged**